plugins/avx512/float/UMESimdVecFloat32_8.h

```diff
@@ -2316,7 +2316,11 @@ namespace SIMD {
             mVec = _mm256_load_ps(raw);
             return *this;
  #else
+ #if defined (WA_GCC_INTR_SUPPORT_7_1)

```

Comment:
Seems like this is dead code: the same macro used as condition in both '#if' and '#else'.

The same comment applies for other changes related to the WA_GCC_INTR_SUPPORT_7_1 macro.